modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -64,13 +55,14 @@ func (suite *KeeperTestSuite) TestEscrowPacketFee() {
 
 		suite.Run(tc.name, func() {
 			suite.SetupTest() // reset
+			suite.coordinator.Setup(suite.path)
 
 			// setup
 			refundAcc = suite.chainA.SenderAccount.GetAddress()
 			ackFee = validCoins
 			receiveFee = validCoins2
 			timeoutFee = validCoins3
-			packetId := &channeltypes.PacketId{ChannelId: validChannelId, PortId: types.PortID, Sequence: uint64(1)}
+			packetId := &channeltypes.PacketId{ChannelId: suite.path.EndpointA.ChannelID, PortId: transfertypes.PortID, Sequence: uint64(1)}
```

Comment:
Switched to suite path's channelID, since it already is setup for us with fee enabled during channel handshake